frontend/package.json

```diff
@@ -35,7 +35,7 @@
     "validateComponents:linux:darwin": "./scripts/validateComponents.sh"
   },
   "dependencies": {
-    "@attivio/suit": "0.0.62",
+    "@attivio/suit": "file:../../suit",
```

Comment:
We can't include a relative dependency like this in the actual Search UI repository. Once there's a build of SUIT that includes the changes you depend on, you can update the dependent version.

FYI: We have started using `npm ci` to pull down the project's dependencies to the node_modules directory instead of `npm install`. This prevents inadvertent/unexpected changes to the versions installed, ensuring that we always are using the same versions of all dependencies as were last built (and tested, etc.). To change dependencies, you should not edit the package.json file directory but instead use the `npm install <library>@<version>` command or `npm remove <library>`, which will update the package.json and package-lock.json files without messing with versions unnecessarily. For example, to set the SUIT dependency to a relative path for local development, use `npm install @attivio/suit@../../suit` (and then, once your SUIT changes have been published to NPM, you can use `npm install @attivio/suit@0.0.63` to set it to the new version, 0.0.63).

Reply:
Fixed

Reply:
Once there's a new build of SUIT with the changes to Searcher to actually use this property, you'll want to create a new Search UI pull request with the updated dependency version.